lab/hub.go

```diff
@@ -179,7 +179,7 @@ func (h *hub) Suspend(ctx context.Context) error {
 		wg.Add(1)
 		go func() {
 			if err := l.Suspend(ctx); err != nil {
-				err = suspendError
+				suspendError = err
```

Comment:
As far as i know these errors is not checked maybe just a log statement

Reply:
hmm those are checked in functions which are calling Suspend function actually. 
for instance here: https://github.com/aau-network-security/haaukins/blob/4b7f5c13ff8ab5cb7ef2ac1b169d57855836a5c9/svcs/guacamole/event.go#L587

Reply:
I actually realized that this is useless, I will remove it as well 
https://github.com/aau-network-security/haaukins/blob/4b7f5c13ff8ab5cb7ef2ac1b169d57855836a5c9/svcs/guacamole/event.go#L586

Reply:
Then it is perfect :ok_hand: